RetailCoder.VBE/Inspections/InspectionBase.cs

```diff
@@ -81,6 +83,28 @@ protected virtual IEnumerable<Declaration> BuiltInDeclarations
             get { return State.AllDeclarations.Where(declaration => declaration.IsBuiltIn); }
         }
 
+        protected bool HasIgnoreAnnotation(VBComponent component, int line)
```

Comment:
I'm pretty sure this duplicates (improves?) existing functionality. I can't remember off the top of my head, IIRC there was a `IsInspectionDisabled` method somewhere. Was it removed (reviewing from phone... painful...)?